x-pack/solutions/security/plugins/security_solution/common/api/detection_engine/rule_management/crud/patch_rule/request_schema_validation.ts

```diff
@@ -55,25 +57,14 @@ const validateTimelineTitle = (rule: PatchRuleRequestBody): string[] => {
   return [];
 };
 
-const validateThreshold = (rule: PatchRuleRequestBody): string[] => {
+const patchRuleExtraValidation = (threshold: Threshold) => {
   const errors: string[] = [];
-  if (rule.type === 'threshold') {
-    if (!rule.threshold) {
-      errors.push('when "type" is "threshold", "threshold" is required');
-    } else {
-      if (
-        rule.threshold.cardinality?.length &&
-        rule.threshold.field.includes(rule.threshold.cardinality[0].field)
-      ) {
-        errors.push('Cardinality of a field that is being aggregated on is always 1');
-      }
-      if (rule.threshold.value <= 0) {
-        errors.push('"threshold.value" has to be bigger than 0');
-      }
-      if (Array.isArray(rule.threshold.field) && rule.threshold.field.length > 3) {
-        errors.push('Number of fields must be 3 or less');
-      }
-    }
+  if (threshold.value !== undefined && threshold.value <= 0) {
+    errors.push('"threshold.value" has to be bigger than 0');
```

Comment:
I'm going off the integration tests we have for both the routes, but aren't the `create` and `update` crud routes doing this same validation of `threshold.value > 0`?  Could this logic just be put in the `validateThresholdBase` util and then named `validateThreshold` without the need to add extra validation?

Reply:
You are right. I think the extra validation is not required because the [schema](https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/security_solution/common/api/detection_engine/model/rule_schema/specific_attributes/threshold_attributes.gen.ts#L42) for `ThresholdValue` already validates that this must be an integer with a min value of 1.